packages/core/src/sdk/product/useProductGalleryQuery.ts

```diff
@@ -78,9 +89,12 @@ export const useProductGalleryQuery = ({
     selectedFacets,
   })
 
-  return useQuery<Query, Variables>(query, localizedVariables, {
+  const fuzzyFacetValue = findFacetValue(selectedFacets, 'fuzzy')
+  const operatorFacetValue = findFacetValue(selectedFacets, 'operator')
+
+  const queryResult = useQuery<Query, Variables>(query, localizedVariables, {
     onSuccess: (data) => {
-      if (data && term) {
+      if (data && term && (fuzzyFacetValue || operatorFacetValue)) {
```

Comment:
It's not related to this PR, as we have already used this logic. But I think we should double-check with the IS team if we should only `sendAnalyticsEvent` with `intelligent_search_query` in the Search Page as it's now OR in both pages the Search Page and PLP.

Now, the event is being triggered only on the Search page because we have the `term` there, but not in PLP.

|PLP|Search|
|-|-|
|<img width="400" alt="Screenshot 2024-11-19 at 18 01 59" src="https://github.com/user-attachments/assets/0725dfbd-f37d-4489-a091-1f5fdcb8a73b">|<img width="400" alt="Screenshot 2024-11-19 at 18 02 23" src="https://github.com/user-attachments/assets/a409af8d-9520-4d6a-8761-be7015ec601c">|

Reply:
Good point! I’ve raised it in this [thread](https://vtex.slack.com/archives/CP6S9HP46/p1728587254285979) so we can address it later.